[2024-11-09T13:19:48.185+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-11-09T13:19:48.208+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pipeline.insert_data_into_data_warehouse manual__2024-11-09T13:19:44.033179+00:00 [queued]>
[2024-11-09T13:19:48.215+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pipeline.insert_data_into_data_warehouse manual__2024-11-09T13:19:44.033179+00:00 [queued]>
[2024-11-09T13:19:48.215+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-11-09T13:19:48.227+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): insert_data_into_data_warehouse> on 2024-11-09 13:19:44.033179+00:00
[2024-11-09T13:19:48.231+0000] {standard_task_runner.py:64} INFO - Started process 196 to run task
[2024-11-09T13:19:48.234+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'pipeline', 'insert_data_into_data_warehouse', 'manual__2024-11-09T13:19:44.033179+00:00', '--job-id', '266', '--raw', '--subdir', 'DAGS_FOLDER/pipline.py', '--cfg-path', '/tmp/tmps67dpi1k']
[2024-11-09T13:19:48.236+0000] {standard_task_runner.py:91} INFO - Job 266: Subtask insert_data_into_data_warehouse
[2024-11-09T13:19:48.276+0000] {task_command.py:426} INFO - Running <TaskInstance: pipeline.insert_data_into_data_warehouse manual__2024-11-09T13:19:44.033179+00:00 [running]> on host 66de996f25f2
[2024-11-09T13:19:48.377+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pipeline' AIRFLOW_CTX_TASK_ID='insert_data_into_data_warehouse' AIRFLOW_CTX_EXECUTION_DATE='2024-11-09T13:19:44.033179+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-09T13:19:44.033179+00:00'
[2024-11-09T13:19:48.378+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-11-09T13:19:48.385+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.411+0000] {logging_mixin.py:188} INFO - Erreur dans la création ou récupération du journal : string indices must be integers, not 'str'
[2024-11-09T13:19:48.412+0000] {logging_mixin.py:188} INFO - Journal Published in: IEEE Transactions on Technology and Society ( Volume: 3, Issue: 4, December 2022) non inséré, arrêt de l'insertion.
[2024-11-09T13:19:48.417+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.428+0000] {logging_mixin.py:188} INFO - Erreur dans la création ou récupération du journal : string indices must be integers, not 'str'
[2024-11-09T13:19:48.429+0000] {logging_mixin.py:188} INFO - Journal Published in: IEEE Transactions on Technology and Society ( Volume: 4, Issue: 1, March 2023) non inséré, arrêt de l'insertion.
[2024-11-09T13:19:48.434+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.492+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication The Generative AI Landscape in Education: Mapping the Terrain of Opportunities, Challenges, and Student Perception: time data 'Date of Publication: 16 September 2024' does not match format '%d %B %Y'
[2024-11-09T13:19:48.497+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.510+0000] {logging_mixin.py:188} INFO - Erreur dans la création ou récupération du journal : string indices must be integers, not 'str'
[2024-11-09T13:19:48.511+0000] {logging_mixin.py:188} INFO - Journal Published in: IEEE Computer Architecture Letters ( Volume: 23, Issue: 2, July-Dec. 2024) non inséré, arrêt de l'insertion.
[2024-11-09T13:19:48.516+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.529+0000] {logging_mixin.py:188} INFO - Erreur dans la création ou récupération du journal : string indices must be integers, not 'str'
[2024-11-09T13:19:48.530+0000] {logging_mixin.py:188} INFO - Journal Published in: IEEE Embedded Systems Letters ( Volume: 14, Issue: 3, September 2022) non inséré, arrêt de l'insertion.
[2024-11-09T13:19:48.535+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.557+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication A Survey of Real-Time Strategy Game AI Research and Competition in StarCraft: time data 'Date of Publication: 18 October 2013' does not match format '%d %B %Y'
[2024-11-09T13:19:48.563+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.587+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication Unraveling ML Models of Emotion With NOVA: Multi-Level Explainable AI for Non-Experts: time data 'Date of Publication: 09 December 2020' does not match format '%d %B %Y'
[2024-11-09T13:19:48.593+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.618+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication AIS-PVT: Long-Time AIS Data Assisted Pyramid Vision Transformer for Sea-Land Segmentation in Dual-Polarization SAR Imagery: time data 'Date of Publication: 26 August 2024' does not match format '%d %B %Y'
[2024-11-09T13:19:48.624+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.643+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication Explainable AI for Cheating Detection and Churn Prediction in Online Games: time data 'Date of Publication: 10 May 2022' does not match format '%d %B %Y'
[2024-11-09T13:19:48.649+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.671+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication Towards Measuring Fairness in AI: The Casual Conversations Dataset: time data 'Date of Publication: 03 December 2021' does not match format '%d %B %Y'
[2024-11-09T13:19:48.676+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.696+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication Personas for Artificial Intelligence (AI) an Open Source Toolbox: time data 'Date of Publication: 25 February 2022' does not match format '%d %B %Y'
[2024-11-09T13:19:48.702+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.726+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication A Game AI Competition to Foster Collaborative AI Research and Development: time data 'Date of Publication: 15 September 2020' does not match format '%d %B %Y'
[2024-11-09T13:19:48.732+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.744+0000] {logging_mixin.py:188} INFO - Erreur dans la création ou récupération du journal : string indices must be integers, not 'str'
[2024-11-09T13:19:48.745+0000] {logging_mixin.py:188} INFO - Journal Published in: IEEE Transactions on Technology and Society ( Volume: 5, Issue: 3, September 2024) non inséré, arrêt de l'insertion.
[2024-11-09T13:19:48.750+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.764+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication XAI-IoT: An Explainable AI Framework for Enhancing Anomaly Detection in IoT Systems: time data 'Date of Publication: 17 May 2024' does not match format '%d %B %Y'
[2024-11-09T13:19:48.770+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.783+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication Human-Centered AI in Smart Farming: Toward Agriculture 5.0: time data 'Date of Publication: 30 April 2024' does not match format '%d %B %Y'
[2024-11-09T13:19:48.790+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.812+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication An Area-Efficient Systolic Array Redundancy Architecture for Reliable AI Accelerator: time data 'Date of Publication: 08 July 2024' does not match format '%d %B %Y'
[2024-11-09T13:19:48.818+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.845+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication Unleashing the Power of Edge-Cloud Generative AI in Mobile Networks: A Survey of AIGC Services: time data 'Date of Publication: 12 January 2024' does not match format '%d %B %Y'
[2024-11-09T13:19:48.851+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.876+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication Exploring the Effect of Generative AI on Social Sustainability Through Integrating AI Attributes, TPB, and T-EESST: A Deep Learning-Based Hybrid SEM-ANN Approach: time data 'Date of Publication: 10 September 2024' does not match format '%d %B %Y'
[2024-11-09T13:19:48.883+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.906+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication The Ethics of AI in Games: time data 'Date of Publication: 16 May 2023' does not match format '%d %B %Y'
[2024-11-09T13:19:48.911+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.921+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication E-XAI: Evaluating Black-Box Explainable AI Frameworks for Network Intrusion Detection: time data 'Date of Publication: 12 February 2024' does not match format '%d %B %Y'
[2024-11-09T13:19:48.927+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.937+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication Demystifying Defects: Federated Learning and Explainable AI for Semiconductor Fault Detection: time data 'Date of Publication: 08 July 2024' does not match format '%d %B %Y'
[2024-11-09T13:19:48.943+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.954+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication Developing a Transparent Diagnosis Model for Diabetic Retinopathy Using Explainable AI: time data 'Date of Publication: 07 October 2024' does not match format '%d %B %Y'
[2024-11-09T13:19:48.960+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:48.973+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication AI and Security, From an Information Security and Risk Manager Standpoint: time data 'Date of Publication: 31 May 2024' does not match format '%d %B %Y'
[2024-11-09T13:19:48.978+0000] {base.py:84} INFO - Using connection ID 'mysql_default' for task execution.
[2024-11-09T13:19:49.000+0000] {logging_mixin.py:188} INFO - Erreur lors de l'insertion de la publication Empirical Evaluations of Framework for Adaptive Trust Calibration in Human-AI Cooperation: time data 'Date of Publication: 04 December 2020' does not match format '%d %B %Y'
[2024-11-09T13:19:49.002+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-11-09T13:19:49.003+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-11-09T13:19:49.015+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=pipeline, task_id=insert_data_into_data_warehouse, run_id=manual__2024-11-09T13:19:44.033179+00:00, execution_date=20241109T131944, start_date=20241109T131948, end_date=20241109T131949
[2024-11-09T13:19:49.049+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-11-09T13:19:49.073+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-11-09T13:19:49.076+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
